lib/classes/PluginManager.js

```diff
@@ -309,6 +310,41 @@ class PluginManager {
     });
   }
 
+  loadVariableResolvers(pluginInstance) {
+    const pluginName = pluginInstance.constructor.name;
+    for (const [variablePrefix, resolverOrOptions] of _.entries(
+      pluginInstance.variableResolvers || {}
+    )) {
+      let options = resolverOrOptions;
+      if (_.isFunction(resolverOrOptions)) {
+        options = {
+          resolver: resolverOrOptions,
+        };
+      }
+      if (!variablePrefix.match || !variablePrefix.match(/[0-9a-zA-Z_-]/)) {
```

Comment:
`variablePrefix` is guaranteed to be string, and string in all cases will have `match` method